### ISSUE TITLE
control RGB bulbs

### ISSUE THREAD

**Issue by thunder-62** [nidayand/node-red-contrib-tradfri#12]
it seems like the "TRADFRI bulb E27 CWS opal 600lm" color bulb can't be controlled with the current release.

Would it be possible to support these bulbs as well?

Thank you

**Comment by nidayand** [comment]
In the new version 1.1.0 support for colors is supported. Please test it

**Comment by thunder-62** [comment]
tested it and it works.  Do you know whether the bulbs only support specific RGB patterns or can I feed in every RGB combination as a hex code?

Thank you,
Uwe

**Comment by nidayand** [comment]
Hey, a fairly new update includes shortnames for RGB values as specified in the argon library :

color (String) - [hex color value, 'cool', 'normal', 'warm'] - color code (HEX - e.g. 'ffffff' or focus, cool, everyday, normal, relax, warm, cold sky, cool daylight, cool white, sunrise, warm white, warm glow, candlelight, warm amber, peach, dark peach, saturated red, pink, light pink, saturated pink, light purple, saturated purple, blue, light blue, lime, yellow)

**Comment by svelog** [comment]
Hi, I tested the new presets and they work great.

Now I wanted to use the HEX color code and realized that this is not working for every color code. For example “ffffff” changes the color to “f1e0b5”. I think this happpens every time when I call a HEX color code which is not part of the preset-colormap. 

In addition I checked the color code with the get-node. As long as I chose a color preset I get a good HEX color code. But when I change the color via Homekit and use afterwards the get-node I get color: “0”.
